website/docs/crud.md

```diff
@@ -0,0 +1,294 @@
+---
+id: crud
+title: CRUD Handler
+sidebar_label: CRUD Handler
+---
+
+## CRUD
+In `Core-Lib`, we have classes that can handle CRUD  ( Create, Read, Update, Delete ) for us.
+They are as follows
+
+
+
+### CRUD
+This class is used to initialize `CRUD` with database objects, rule validators and database handlers. It is the Base

```

Comment:
add this as well: 
`CRUD` is a abstract base class for all types of `CRUDDataAccess`. 

Use grammerly